tests/components/jellyfin/test_config_flow.py

```diff
@@ -7,7 +7,7 @@
 from homeassistant.core import HomeAssistant
 
 from . import async_load_json_fixture
-from .const import TEST_PASSWORD, TEST_URL, TEST_USERNAME
+from .const import TEST_PASSWORD, TEST_URL, TEST_USERNAME, USER_INPUT
 
 from tests.common import MockConfigEntry
```

Comment:
For config-flow tests, we want to ensure that async_setup_entry is never used, and it's quite easy to miss (you missed it on `test_reauth`).
I suggest that you add this line to ensure it is used on all tests within the file.

```suggestion
from tests.common import MockConfigEntry

pytestmark = pytest.mark.usefixtures("mock_setup_entry")
```

Reply:
merge solved this